deeprank_gnn/NeuralNet.py

```diff
@@ -383,17 +442,16 @@ def eval(self, loader: DataLoader, epoch_number: int, pass_name: str) -> float:
         outputs = []
         entry_names = []
 
-        batch_count = len(loader)
+        # batch_count = len(loader)
```

Comment:
Same here. I am asking all this confirmation just to be sure that these unused variables were actually not supposed to be used.

Reply:
Let's remove batch count.